src/lib/dacc.js

```diff
@@ -199,7 +199,7 @@ export const sendMeasuresForAccount = async (client, account) => {
  */
 export const runDACCService = async client => {
   const settings = await client.queryAll(buildSettingsQuery().definition)
-  if (!settings || !settings[0].allowSendDataToDacc) {
+  if (!settings || !settings[0] || !settings[0].allowSendDataToDacc) {
```

Comment:
`if (!settings?.[0]?.allowSendDataToDacc)` should be enough?

Reply:
Yeah, that also works.